config/.eslintrc.json

```diff
@@ -0,0 +1,150 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+        "@typescript-eslint",
+        "@typescript-eslint/tslint"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "overrides": [
+        {
+            "files": [
+                "*test.ts",
+                "**/test/**/*.ts"
+            ],
+            "rules": {
+                "no-unused-expressions": "off"
+            }
+        }
+    ],
+    "rules": {
+        "arrow-body-style": [
+            "error",
+            "as-needed",
+            {
+                "requireReturnForObjectLiteral": true
+            }
+        ],
+        "curly": [
+            "error",
+            "multi-line"
```

Comment:
Even though it's a bit stricter than Google TS style (doesn't allow braceless single line if), I vote for `"all"`.

Reply:
I'd be fine with "all". I'm leaving it unresolved for a couple of days to see if anyone objects.

Reply:
Firestore is a fan of "all" too, thought this will probably create some violations since currently we allow no braces iff the statement is on the same line as the if, e.g.:
```
if (result.skip) continue;
```

Looks like eslint doesn't have a similar configuration, so we'll probably lose that... but I think that's okay.

Reply:
Changed to `all`